modules/installation-options.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='installation-options-{context}']
+= Installation options
+
+{product-title} offers two installation options: fully-managed infrastructure and bring your own
+infrastructure. In version 4.0, the fully-managed option allows you to install a
+cluster in Amazon Web Services (AWS) that runs on Red Hat CoreOS nodes. If you want to
+use any other cloud or install your cluster on-premise, use the bring your own
+infrastructure option to install your cluster on existing Red Hat Enterprise 
+Linux (RHEL) hosts.
+
+The fully-managed option offers full-stack automation to:
+
+* Manage compute
+* Manage operating system (RHCOS)
+* Manage control plane
+* Manage nodes
+
+The bring your own infrastructure option, you have more responsibilities.
+You must provide the hosts and update RHEL on them. {product-title} provides:
+
+* Managed control plane
+* Ansible to manage kubelet and runtime
+
+
+With both installation types, installation and upgrade both use a controller
+that constantly reconciles component versions as if it were any other Kubernetes
```

Comment:
Not really true, also I would not use controller in our docs if we can use “operator” instead.  We need to nuance this a bit more.